ui/nuxeo-document-preview.js

```diff
@@ -374,6 +376,14 @@ import './viewers/nuxeo-video-viewer.js';
     get _isVisible() {
       return Boolean(this.offsetWidth || this.offsetHeight);
     }
+
+    _computeCodecs(codecs) {
+      if (!codecs) {
+        return '';
+      }
+      const exp = /\w+\s\(\w+\)\s\((.+)\s\/(\s\w+\))/g;
```

Comment:
I also thing that (.+) might be to permissive, as it will include anything like (). I think tat we could be a bit more explicit to what we might be hoping to find, like \w, plus maybe dots and dashes? e.g.:
```suggestion
      const exp = /\w+\s\(\w+\)\s\(([a-zA-Z0-9_.-]+)\s\/\s\w+\)/g;
```